modules/thermal_hydraulics/test/tests/closures/THM_1phase/thm1phase.i

```diff
@@ -0,0 +1,151 @@
+# Using no closure option and setting up custom materials that computes f_D and Hw.
+# In this case, these custom materials are computing just constant values
+
+k = 0.56361
+mu = 8.84e-05
+D = 0.1
+A = '${fparse (1./4.)*pi*D^2}'
+P_hf = '${fparse pi*D}'
+D_h = '${fparse 4*A/P_hf}'
+mdot = 0.04
+
+[GlobalParams]
+  gravity_vector = '0 0 0'
+
+  initial_vel = 0.003
+  initial_p = 1e5
+  initial_T = 300
+
+  D_h = ${D_h}
+  A = ${A}
+  P_hf = ${P_hf}
+
+  m_dot = ${mdot}
+  k = ${k}
+  mu = ${mu}
+  PoD = 1.1
+  heat_transfer_geom = "hex_rod_bundle"
+  pipe_location = "interior"
+
+  closures = no_closures
+  execute_on = 'initial timestep_begin'
+[]
+
+[FluidProperties]
+  [water]
+    type = StiffenedGasFluidProperties
+    gamma = 2.35
+    cv = 1816.0
+    q = -1.167e6
+    p_inf = 1.0e9
+    q_prime = 0
+  []
+[]
+
+[Closures]
+  [no_closures]
+    type = Closures1PhaseTHM
+    wall_htc_closure = lyon
+    wall_ff_closure = cheng_todreas
+  []
+[]
+
+[Materials]
+  [velocity]
```

Comment:
these materials are not needed

Reply:
Yes, I forgot to take them out, I was using them to verificative the HW and f values by hand :3

Reply:
done